api/api/versions_api.go

```diff
@@ -115,6 +117,10 @@ func (c *VersionsController) CreateVersion(r *http.Request, vars map[string]stri
 		return BadRequest("Valid label key/values must be 63 characters or less and must be empty or begin and end with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between.")
 	}
 
+	if versionPost.PythonVersion == "" { // if using older sdk, this will be blank
+		versionPost.PythonVersion = DEFAULT_PYTHON_VERSION
+	}
+
```

Comment:
I think it will be an overkill, especually since the value in this case is not determined by users but by our code. There should be no concern about ourselves giving invalid version format. 😅